motorSupportApp/Db/simple.db

```diff
@@ -0,0 +1,263 @@
+record(bo, "$(P)SIM")
```

Comment:
I don't quite understand why this file has been copy-pasted to here. Maybe I'm missing something? If it's required can we just put it in a support module common to both simple IOC and this

Reply:
Apologies, left over from a previous test.